repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -95,11 +95,14 @@
             </xs:annotation>
         </xs:attribute>
 
-        <!-- Check to make sure we really need this to be a Name -->
+        <!-- TODO for v9.0.0.0: The DeploymentConfigurationBase element's via attribute should be removed. -->
         <xs:attribute name="via" type="xs:string" use="optional">
```

Comment:
> A new element will exist that is mutually exclusive of the existing attribute

I think this is missing an assertion or something that enforces mutual exclusivity.